repository/pom.xml

```diff
@@ -83,6 +83,7 @@
         <dependency>
             <groupId>commons-fileupload</groupId>
             <artifactId>commons-fileupload</artifactId>
+            <version>1.5</version>
```

Comment:
Unnecessary as this is handled by the `dependencyManagement` section of the pom.

Reply:
Revert 👍